zhaquirks/sonoff/snzb02d.py

```diff
@@ -63,6 +63,7 @@ class AttributeDefs(BaseAttributeDefs):
 
 (
     QuirkBuilder("SONOFF", "SNZB-02D")
+    .add_model("SONOFF", "SNZB-02DR2")

```

Comment:
Please use `applies_to`:
```suggestion
    .applies_to("SONOFF", "SNZB-02DR2")
```